files/usr/share/cinnamon/cinnamon-settings/modules/cs_default.py

```diff
@@ -552,9 +569,15 @@ def on_module_selected(self):
             widget = SettingsWidget()
             button = DefaultTerminalButton()
             label = MnemonicLabel(_("Te_rminal"), button)
-            size_group.add_widget(button)
+            entry = TerminalExecArgEntry()
+
+            box = Gtk.Box()
```

Comment:
Try making this a vertical box - I think it'll be cleaner keeping the picker button the same size as the rest on the page, with the entry below.

Reply:
Done! Thanks for the suggestion.